pkg/asset/cluster/aws/aws.go

```diff
@@ -207,17 +234,25 @@ func tagSharedIAMProfiles(ctx context.Context, clusterID string, installConfig *
 
 	logrus.Debugf("Tagging shared instance profiles: %v", sets.List(iamProfileNames))
 
-	session, err := installConfig.AWS.Session(ctx)
+	cfg, err := configv2.LoadDefaultConfig(context.TODO(), configv2.WithRegion(installConfig.Config.Platform.AWS.Region))
```

Comment:
```suggestion
	cfg, err := configv2.LoadDefaultConfig(ctx, configv2.WithRegion(installConfig.Config.Platform.AWS.Region))
```

Same as above :smile: